packages/database/versioning.go

```diff
@@ -0,0 +1,49 @@
+package database
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/iotaledger/hive.go/database"
+)
+
+const (
+	// defines which version of the database schema this version of GoShimmer supports.
+	// everytime there's a breaking change regarding the stored data, this version flag should be adjusted.
+	DBVersion byte = 1
```

Comment:
```suggestion
	DBVersion = 1
```
Untyped constant should be preferred.